### DISCUSSION TITLE
antd1 和 antd4  css样式差别大吗

### DISCUSSION THREAD

**Discussion by ghost** [ant-design/ant-design#33230] [Q&A]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
有个项目既有ant1 又想加入antd4, 如果两者样式没大改动的话, 那我就不用加入antd4的样式了吧

### What does the proposed API look like?
有个项目既有ant1 又想加入antd4, 如果两者样式没大改动的话, 那我就不用加入antd4的样式了吧

**Comment by xrkffgg** [comment]
正好把 antd1 升级了

**Comment by ghost** [comment]
> 

现状是不能升级
